front-end/e-learning/src/components/Login.css

```
@@ -0,0 +1,7 @@
+.container{
+}
+
+.column{
+    background-color: black;
+    max-width: 50%;
+}
```

Remove `container` if not needed.
Remember to always put space before `{`


```suggestion
.column {
    background-color: black;
    max-width: 50%;
}
```